frontend/apps/reader/modules/readersearch.lua

```diff
@@ -83,6 +83,15 @@ function ReaderSearch:addToMainMenu(menu_items)
     menu_items.fulltext_search_settings = {
         text = _("Fulltext search settings"),
         sub_item_table = {
+            {
+                text = _("Search highlighted text: find all"),
```

Comment:
May be `[ ] Show all results (or result list) on text selection` ?
I guess a `help_text` could be:
`When invoked after text selection, show a list with all results instead of highlighting matches in book pages.`